Server-Protocol-Buffer/src/main/proto/applicationConfiguration.proto

```diff
@@ -13,7 +13,9 @@ message ApplicationConfiguration {
 
     ApplicationVersionConfiguration appVersion = 5;
 
-    repeated string supportedCountries = 6;
+    AppFeatures appFeatures = 6;
+
+    repeated string supportedCountries = 7;
```

Comment:
Already implemented in https://github.com/corona-warn-app/cwa-app-android/pull/1202